File: ios/RNEventEmitter.m
//
//  RNEventEmitter.m
//  Readerly
//
//  Created by Monisa Hassan on 8/31/22.
//

#import <Foundation/Foundation.h>
#import <React/RCTBridgeModule.h>
#import <React/RCTEventEmitter.h>

@interface RCT_EXTERN_MODULE(RNEventEmitter, RCTEventEmitter)
  RCT_EXTERN_METHOD(supportedEvents)
@end

File: ios/RNEventEmitter.swift
//
//  RNEventEmitter.swift
//  Readerly
//
//  Created by Monisa Hassan on 8/31/22.
//

import Foundation
@objc(RNEventEmitter)
open class RNEventEmitter: RCTEventEmitter {

  public static var emitter: RCTEventEmitter!

  override init() {
    super.init()
    RNEventEmitter.emitter = self
  }

  open override func supportedEvents() -> [String] {
    ["onCPress","onBPress","onSPress", "onCBackPress","onBBackPress","onSBackPress"]
  }
}
